File: 002_AgentesDeResolucaoDeProblemas/src/Mapa.java
import java.util.Map;

public class Mapa {
    private Map<Estado, Estado[]> cidades;

    public Mapa (Map<Estado, Estado[]> cidades) {
        this.cidades = cidades;
    }

    public Map<Estado, Estado[]> getCidades() {
        return this.cidades;
    }
}

File: 002_AgentesDeResolucaoDeProblemas/Mapa.py
class Mapa:
    def __init__(self, cidades):
        self.cidades = cidades

    def __str__(self):
        os = ""

        for dd in self.cidades:
            os += dd.nome + " ->"
            for d in self.cidades[dd]:
                os += f"{d[0].nome} = {d[1]}km, "
            os += "\n"

        return os